domains/bypassrn.json

```diff
@@ -0,0 +1,13 @@
+{
+        "owner": {
+           "username": "Gauravisdev",
+           "email": "",
+           "discord": "1119494339735867422",
+           "OWL": "eyJlbmMiOiJBMTI4Q0JDLUhTMjU2IiwiYWxnIjoiUlNBLU9BRVAiLCJraWQiOiJaa1VsRmRqVThiUEstLXVVM2JJR09PVHFYYVFFS1ZINFVXOW53MTR6WTJnIn0.pzBM0Ie4fgb-zJ_0jQc_wvhPQXn1Smlfqby_KW4z3xZOsk3FomBiCHqFCcq6nToxqTYsz_aUkZLXwaJjVZ30Z-vKAN1ttgFCnexwxAGwreKsT9gyST1k1icHg3Uay5KJPUNzGeTvd4kSvJ6z76aYu-aU2YEL-bnUAvfCKlzWVlmFHxoVKJ56AZcr9kY8oCegjPS6FNwewO1pF7ie78SEo1l-fY-Np-PWvkEInGJRCGcprpbhNQxaqBfkGItnGkeQV4jr40L3UokvXjbyPbhYLPJROb-AxMZAWvR8ryVQjaSDhTevdZ1xsWJxGBVMKxum3I7FYH7A5Uz_TPrdbKtvmA.mebJqhTod9St7BqPCw398A.v11ADOcJnwiV8_HbsqhVZcFlvsWUTKJEnDyZb0W7p3ezoF5nMrDSWwFiWAlwDk0xXkWE98HGG7dKVGCt4gidQ10Ad5U2-1hmqsVTKAJAj48.Zr6KvC7GxLy7BJ9LDCQoqA"
+        },
+    
+        "record": {
+            "CNAME": "https://gaurav-sto.onrender.com/"
```

Comment:
CNAME can not contain `https://` or `/`

Reply:
oh yeah I forgot sorry 
changes have been made